### PR TITLE
Add support for return SELinux and other MAC Labels

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: test/format.bats [hunk 1]
```diff
@@ -203,3 +203,9 @@
 	[[ ${lines[0]} =~ "VSZ" ]]
 	[[ ${lines[0]} =~ "SECCOMP" ]]
 }
+
+@test "LABEL header" {
+	run ./bin/psgo -format "label"
```
Comment (vrothberg) [inline test/format.bats]: Could you add it to test above as well? Just for consistency.

**Comment by rhatdan** [comment]
@vrothberg How is the code supposed to work?

**Comment by vrothberg** [comment]
> @vrothberg How is the code supposed to work?

What do you mean precisely?

Travis will still need https://github.com/containers/psgo/blob/master/test/list.bats to be extended with the new `label` flag.  You can run a `make test` locally to spin up the unit and integration tests.

**Comment by rhatdan** [comment]
It works locally, I am wondering if travis did not catch the previous push and continued testing the old code.

**Comment by vrothberg** [comment]
I don't see a change to `test/list.bats` in the latest commit. Are you sure you added the change to the commit?

**Comment by rhatdan** [comment]
There was no change. I was just a repush.
I do not see this failing locally, this might be failing because ubuntu is giving an error NOEXIST so label is not being printed out.

**Comment by vrothberg** [comment]
I'll look into it tomorrow. I am currently hurrying to get the change ready for libpod.

**Comment by rhatdan** [comment]
@vrothberg I think I got it fixed.  Added a check for SELinux being enabled, which should turn off the test on Ubuntu.

**Comment by vrothberg** [comment]
LGTM

**Pull request merged**